apps/web/src/components/templates/TriggerSnippetTabs.tsx

```diff
@@ -10,7 +10,7 @@ const novu = new Novu('<API_KEY>');
 
 novu.trigger('${trigger.identifier?.replace(/'/g, "\\'")}', {
```

Comment:
I think that we only need the change to takes place in cURL on line 35 and the rest need no change - @scopsy  WDYT?

Thanks Tosin!!